README.rst

```diff
@@ -203,46 +280,14 @@ The tracing wrapper creates a span for the wrapper handler. This span has the fo
 |                                  | ‘python-lambda-wrapper’          |
 +----------------------------------+----------------------------------+
 
-Sending custom metric from the Lambda function
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. code:: python
 
-    import signalfx_lambda
+Steps 7 & 8 don't have (and shouldn't) be done by most users. This is rather info for contributors who want to test changes they do in the wrapper and/or publish a new version. As such, I would vote to have them in last separate section with a title "For contributors" or "For maintainers" or just "Additional info". The metrics and dimensions and tags which are now last are probably of interest to most users, but this two steps are not.
 
-    # sending application_performance metric with value 100 and dimension abc:def
-    signalfx_lambda.send_gauge('application_performance', 100, {'abc':'def'})
 
-    # sending counter metric with no dimension
-    signalfx_lambda.send_counter('database_calls', 1)
+Test configurations locally 
```

Comment:
I would just say "Test locally" without "configurations". As a programmer, I wouldn't know what "configurations" are in this context. I think we can also assume that someone who tests this locally know what they do and don't need more info.

Reply:
This has been updated.